packages/salesforcedx-vscode-core/src/conflict/conflictNode.ts

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import * as vscode from 'vscode';
+import { nls } from '../messages';
+
+export type ConflictFile = {
+  remoteLabel: string;
+  fileName: string;
+  relPath: string;
+  localPath: string;
+  remotePath: string;
+};
+
+export class ConflictNode extends vscode.TreeItem {
+  private _children: ConflictNode[];
+  private _parent: ConflictNode | undefined;
+  protected _conflict: ConflictFile | undefined;
+
+  constructor(
+    label: string,
+    collapsibleState: vscode.TreeItemCollapsibleState,
+    parent?: ConflictNode
+  ) {
+    super(label, collapsibleState);
+    this._children = [];
+    this._parent = parent;
+  }
+
+  get conflict() {
+    return this._conflict;
+  }
+
+  get parent() {
+    return this._parent;
+  }
+
+  get children() {
+    return this._children;
+  }
+
+  get tooltip() {
+    return this._conflict ? this._conflict.relPath : this.label;
+  }
+}
+
+export class ConflictFileNode extends ConflictNode {
+  constructor(conflict: ConflictFile, parent: ConflictNode) {
+    super(conflict.fileName, vscode.TreeItemCollapsibleState.None, parent);
+    this._conflict = conflict;
+  }
+
+  public attachCommands() {
+    this.contextValue = 'conflict-actions';
+    this.command = {
+      title: 'Compare',
```

Comment:
This should be a label.

Reply:
I can make it a label, but I never saw it displayed anywhere.